### ISSUE TITLE
onChangeIndex been constantly called

### ISSUE THREAD

**Issue by unmec** [gusgard/react-native-swiper-flatlist#100]
Starting from version ```3.0.0```, I realize that ```onChangeIndex``` is been constantly called, while ```2.0.7``` works as expected:

```
<SwiperFlatList
  onChangeIndex={(event) => {
    console.log(event)
  }}
>
  {someChildren}
</SwiperFlatList>
```

Xcode will log the event non-stop.

React Native version:
0.63.4

Xcode version:
12.3

MacOS version:
10.15.7

**Comment by gusgard** [comment]
Thanks for reaching out, I will take a look at it and let you know soon

**Comment by gusgard** [comment]
Released with 3.0.7

**Comment by unmec** [comment]
Thanks!
